jobs/nightly-samba-rpm-builds.yml

```diff
@@ -0,0 +1,60 @@
+- job:
+    name: gluster_nightly-samba-rpm-builds
+    node: gluster
+    description: Build Samba RPMs.
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: https://github.com/gluster/centosci.git
+        branches:
+        - origin/master
+
+    properties:
+    - github:
+        # TODO: change the repo to final location
+        url: https://github.com/obnoxxx/samba-integration
+    - build-discarder:
+        days-to-keep: 7
+        artifact-days-to-keep: 7
+
+    triggers:
+    - timed: "H 0 * * *"
+    # TODO: enable PR checking on the repo once in final location
+    #- github-pull-request:
+    #    admin-list:
+    #    - obnoxxx
+    #    - gd
+    #    - anoopcps9
+    #    - spuiuk
+    #    - nixpanic
+    #    cron: H/5 * * * *
+    #    status-context: centos-ci
+
+    builders:
+    - shell: !include-raw: scripts/common/get-node.sh
+    - shell: !include-raw: scripts/common/rsync.sh
+    - shell: jobs/scripts/common/bootstrap.sh $WORKSPACE/jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh
```

Comment:
Does this script exist somewhere already? You probably should push it together with this PR.

Reply:
It is, isn't it?!

Reply:
Strange, after reloading the page it is!?